.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+node_js:
+  - "node"
+  - "0.10"
```

Comment:
Helps to ensure the project builds on a wide range of Node.js versions by targeting both 0.10.x and latest.